src/app_recovery.js

```diff
@@ -58,8 +63,14 @@ class AppModeApp extends LitElement {
     this.isLoggedIn = false;
     this.activeStepNumber = 0;
     this.setupState = null;
+    this.isFirstTimeSetup = false;
+
     bindToClass(renderChunks, this);
     this.context = new StoreSubscriber(this, store);
+
+    this.reflectorToken = Math.random().toString(36).substring(2, 14);
+    // this.reflectorHost = 'https://reflector.dogebox.org'
+    this.reflectorHost = "http://127.0.0.1:9999"
```

Comment:
I don't really know how to deal with this. Normally I'd use build/environment args but as we're just serving straight static files that won't work. Maybe we put into the debug config?

Reply:
done